javatests/google/registry/export/ExportConstantsTest.java

```diff
@@ -82,12 +82,15 @@ public void testReportingKinds_areSubsetOfBackupKinds() throws Exception {
     assertThat(ExportConstants.getBackupKinds()).containsAllIn(ExportConstants.getReportingKinds());
   }
 
-  @Test
-  public void testReportingEntityClasses_areAllBaseEntityClasses() throws Exception {
-    for (Class<? extends ImmutableObject> clazz : ExportConstants.REPORTING_ENTITY_CLASSES) {
-      assertThat(clazz.isAnnotationPresent(Entity.class))
-          .named(String.format("class %s is an @Entity", clazz.getSimpleName()))
-          .isTrue();
-    }
-  }
+  @Test public void testReportingKinds_count() throws Exception { 
+    assertThat(ExportConstants.getReportingKinds().size()).isEqualTo(28); 
```

Comment:
This test doesn't seem useful to me.  What's the point?  It's just going to randomly break in the future if new exported kinds are added, and someone will have to increment the number.

What we really want is a golden file test, call it exported_kinds.txt, that works just like the backup_kinds.txt test.